linux_os/guide/services/cron_and_at/service_atd_disabled/tests/service_disabled.pass.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_ospp,xccdf_org.ssgproject.content_profile_standard
```

Comment:
Rule `service_atd_disabled` is not selected in any `ospp` profile, why have it here?